backend/src/utils/http.ts

```diff
@@ -0,0 +1,15 @@
+import { Request } from 'express'
+import { ByteRange } from '@autonomys/file-caching'
+
+export const getByteRange = (req: Request): ByteRange | undefined => {
+  const byteRange = req.headers['content-range']
+  if (byteRange == null) {
+    return undefined
+  }
```

Comment:
It would be useful to do some further validation (formatting, negative numbers, start > end, etc.).